core/ecschema-metadata/test/UnitConversion/TestUnitsA.ecschema.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ECSchema schemaName="UnitsA" alias="uA" version="01.00.05" description="Creating another schema to test unit conversion across schemas" displayLabel="Units Testing" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECSchemaReference name="UnitsB" version="01.00.05" alias="uB"/>
+    <ECSchemaReference name="UnitsC" version="01.00.05" alias="uC"/>
+    <UnitSystem typeName="SI" />
+    <Phenomenon typeName="LENGTH" definition="LENGTH" displayLabel="Length" />
+    <Phenomenon typeName="NUMBER" definition="NUMBER" displayLabel="Number" />
+    <Unit typeName="M" phenomenon="LENGTH" unitSystem="SI" definition="M" displayLabel="m" />
+
+    <Unit typeName="unitA1" phenomenon="LENGTH" unitSystem="SI" definition="UnitsB:unitB1*UnitsC:unitC1"/>
+    <Unit typeName="unitA2" phenomenon="LENGTH" unitSystem="SI" definition="UnitsA:unitA3*UnitsB:unitB2"/>
+    <Unit typeName="unitA3" phenomenon="LENGTH" unitSystem="SI" definition="M" numerator="10.0"/>
+    <Unit typeName="unitA4" phenomenon="LENGTH" unitSystem="SI" definition="unitA5*UnitsB:unitB5"/>
+    <Unit typeName="unitA5" phenomenon="LENGTH" unitSystem="SI" definition="UnitsB:unitB5"/>
+    <Unit typeName="unitA6" phenomenon="LENGTH" unitSystem="SI" definition="unitA5(-1)*UnitsB:unitB5(2)"/>
+    <Unit typeName="unitA7" phenomenon="LENGTH" unitSystem="SI" definition="uB:unitB8*uC:unitC1"/>
+    <Unit typeName="unitA8" phenomenon="LENGTH" unitSystem="SI" definition="uA:unitA3*uB:unitB9"/>
+
+    <Unit typeName="unitA9" phenomenon="LENGTH" unitSystem="SI" definition="uB:unitB9"/>
+    <Unit typeName="unitA10" phenomenon="LENGTH" unitSystem="SI" definition="uA:unitA9(-1)*uB:unitB9(2)"/>
+    <Unit typeName="unitA11" phenomenon="LENGTH" unitSystem="SI" definition="uD:unitD1"/>
+    <Unit typeName="unitA12" phenomenon="LENGTH" unitSystem="SI" definition="uB:unitB12"/>
+</ECSchema>
```

Comment:
I want to make sure our test schemas are technically correct ... not just good enough to load today.

So some rules.  A phenomenon and the units in it must have the same dimensional derivation.  That is to say The Phenomenon Length has a definition of LENGTH.  Area has the definition of LENGTH(2)

My suggestion is to take units and phenomena from the actual units schema and split them up into several schemas to make your test cases.  That will help you make valid definitions and I think make it easier to read and review.  You could also choose to make definitions for some of the standard comedic units ... yes they exist.  Look up Smoot.